server/mediaserver.go

```diff
@@ -210,6 +183,13 @@ func gotRTMPStreamHandler(s *LivepeerServer) func(url *url.URL, rtmpStrm stream.
 			return ErrStorage
 		}
 		storage := drivers.NodeStorage.NewSession(string(mid))
+		// Build the source video profile from the RTMP stream.
+		resolution := fmt.Sprintf("%vx%v", rtmpStrm.Width(), rtmpStrm.Height())
+		vProfile := ffmpeg.VideoProfile{
+			Name:       "source",
+			Resolution: resolution,
+			Bitrate:    "4000k", // Fix this
```

Comment:
Is this just a placeholder?

Reply:
Yeah, it was taken from the "old" 720p default, https://github.com/livepeer/lpms/blob/master/ffmpeg/videoprofile.go#L31 . Mostly inaccurate until we have a better way of returning the estimated bitrate.